Gemfile

```diff
@@ -12,4 +12,5 @@ end
 
 gem 'rake'
 gem 'fpm'
+gem 'ostruct'
```

Comment:
Unbundled in Ruby 3.5, so this just prevents an annoying warning for now, and prevents breakage later.